### ISSUE TITLE
[ BUG ] `tar` and `update query` not working in related RTR commands

### ISSUE THREAD

**Issue by rootisareservedword** [CrowdStrike/psfalcon#427]
Command list inconsistencies in Real-time Response functions

**Describe the bug**
In the command lists of the `Invoke-FalconAdminCommand` and `Invoke-FalconResponderCommand` functions:
1. The `update install` command is duplicated.
2. The `update query` command is missing.
3. The `tar` command is missing.

**To Reproduce**
1. Examine the `ValidateSet` attribute of the `$Command` parameter in both `Invoke-FalconAdminCommand` and `Invoke-FalconResponderCommand` functions.

```powershell
# Example for `Invoke-FalconAdminCommand`
(Get-Command Invoke-FalconAdminCommand).Parameters['Command'].Attributes | 
    Where-Object { $_ -is [System.Management.Automation.ValidateSetAttribute] } |
    Select-Object -ExpandProperty ValidValues
```

2. Observe that `update install` is listed twice, while `update query` and `tar` are not present.

**Expected behavior**
1. The `update install` command should appear only once in the `ValidateSet`.
2. The `update query` command should be available.
3. The `tar` command should be available.

**Environment (please complete the following information):**
 - OS: Windows 10
 - PowerShell: 5.1
 - PSFalcon: 2.2.7

**Comment by bk-cs** [comment]
Thank you for catching this! I have corrected this for the next PSFalcon release.

If you'd like to resolve it for your local module before release, you can replace `public\real-time-response.ps1` using the steps outlined below.

```powershell
Import-Module -Name PSFalcon
$ModulePath = (Show-FalconModule).ModulePath
(Invoke-WebRequest -Uri https://raw.githubusercontent.com/CrowdStrike/psfalcon/b603fe7a395982a39aa09222a5ca08c221f62c06/public/real-time-response.ps1 -UseBasicParsing).Content > (Join-Path (Join-Path $ModulePath public) real-time-response.ps1)
```

Please ensure that you close and re-open PowerShell and re-import PSFalcon before testing.

**Comment by rootisareservedword** [comment]
Awesome! Thanks, and glad to help!

**Comment by bk-cs** [comment]
Leaving this issue open for people to apply the fix until the next release.

**Comment by bk-cs** [comment]
This fix has been included in the 2.2.8 release which is now available on GitHub. I'm going to leave this issue open until the PowerShell Gallery release is also available.
